source/develop/release-management/features/storage/pass_discard_from_guest_to_underlying_storage.html.md

```diff
@@ -0,0 +1,234 @@
+---
+authors: ishaby
+wiki_title: Features/PassDiscardFromGuestToUnderlyingStorage
+feature_name: Pass discard from guest to underlying storage
+feature_modules: engine/vdsm/api-model
+feature_status: Development
+---
+
+# Pass discard from guest to underlying storage
+
+## Summary
+
+### How we use thin provisioned disks today
```

Comment:
Who are "we"? I'd phrase this in a passive form - "How thinly provisioned disks are used today".